podcast/2.7.md

```diff
@@ -0,0 +1,717 @@
+
+**Ivar Østhus:**
+And remember that the rules are evaluated in the application and not in the server API. So, we need to actually make sure that SDK does it correctly. So, to do this, we also have created a client specification framework so that we can use that to drive our integration test for all the SDKs to make sure that they evaluate the rules the same way.
+
+**Beyang Liu:**
+Yeah. Makes sense. And I'm sure there's a ton of interesting tech there that I want to dive into in a little bit.
+
+**Beyang Liu:**
+But before we get into that, Egil you mentioned doing some sort of competitive analysis at the very get-go. And I think a lot of people are probably trying to triangulate where Unleash stands with respect to other feature flagging tools out there, like LaunchDarkly or Split or other tools. Can you talk a little bit about what sets Unleash apart? Or why your customers decide to use you over other options?
+
+**Egil Østhus:**
+Sure. Absolutely. And it's a great question. And as you mentioned there, this is a category that is growing. It's definitely an area that is getting a lot of attention at the moment.
+
+**Egil Østhus:**
+Obviously, LaunchDarkly, looking at the funding and I believe also revenue, is the biggest player on the ground. They have been around about the same, as long as we have been around as an open source project, interestingly enough. But they have been more visible and more loud out there.
+
+**Egil Østhus:**
+And I think that's the biggest. Of course, the most obvious one, we are open source and most of the other competitors are not.
+
+**Egil Østhus:**
+I wouldn't say that is not necessarily a differentiator. It gives us some advantages for sure. I mean, just the fact that people are able to really dive into the source code, they are able to see what is happening, they are really getting this pulse of, "is this something that happens in this tool or not?" Quite a few customers, or let's say the developers, are coming back and saying, "We have looked at your open source repo. There is a lot of activity. We like that. We also see that there are contributors to this. This is something we believe in."
+
+**Egil Østhus:**
+And I also believe it provides quite a high level of trust because we know that every developer are able to inspect the code. So, I trust the developer teams to be able to really be top-notch. And they are.
+
+**Egil Østhus:**
+The feedback so far is that this is really, really rock solid.
+
+**Egil Østhus:**
+And actually, a true story, we were talking to some friends of our out of Sweden, Codecentric, I believe is what they are called. They were doing technical depth management and code inspection. And the CTO of the company was investigating our open source code because they wanted to identify some tech depth. And he came back and said, "Well, this is not interesting. This is just too good."
```

Comment:
```suggestion
And actually, a true story, we were talking to some friends of ours out of Sweden, Codecentric, I believe is what they are called. They were doing technical depth management and code inspection. And the CTO of the company was investigating our open source code because they wanted to identify some tech depth. And he came back and said, "Well, this is not interesting. This is just too good."
```

Reply:
Listening back to the audio, "tech depth" here might be "tech debt," but it's really hard to tell.